tests/test_unit/test_validators.py

```diff
@@ -27,46 +27,46 @@ class TestValidators:
         },  # valid input, will generate default names
         {
             "names": ["a", "b"],
-            "array_type": "multi_track_array",
+            "array_type": "multi_individual_array",
             "individual_names_expected_exception": does_not_raise(["a", "b"]),
             "keypoint_names_expected_exception": does_not_raise(["a", "b"]),
         },  # valid input
         {
             "names": ("a", "b"),
-            "array_type": "multi_track_array",
+            "array_type": "multi_individual_array",
             "individual_names_expected_exception": does_not_raise(["a", "b"]),
             "keypoint_names_expected_exception": does_not_raise(["a", "b"]),
         },  # valid input, will be converted to ["a", "b"]
         {
             "names": [1, 2],
-            "array_type": "multi_track_array",
+            "array_type": "multi_individual_array",
             "individual_names_expected_exception": does_not_raise(["1", "2"]),
             "keypoint_names_expected_exception": does_not_raise(["1", "2"]),
         },  # valid input, will be converted to ["1", "2"]
         {
             "names": "a",
-            "array_type": "single_track_array",
+            "array_type": "single_individual_array",
             "individual_names_expected_exception": does_not_raise(["a"]),
             "keypoint_names_expected_exception": pytest.raises(ValueError),
-        },  # single track array with multiple keypoints
+        },  # single individual array with multiple keypoints
         {
             "names": "a",
             "array_type": "single_keypoint_array",
             "individual_names_expected_exception": pytest.raises(ValueError),
             "keypoint_names_expected_exception": does_not_raise(["a"]),
-        },  # single keypoint array with multiple tracks
+        },  # single keypoint array with multiple individuals
         {
             "names": 5,
-            "array_type": "multi_track_array",
+            "array_type": "multi_individual_array",
             "individual_names_expected_exception": pytest.raises(ValueError),
             "keypoint_names_expected_exception": pytest.raises(ValueError),
         },  # invalid input
     ]
 
-    @pytest.fixture(params=pose_tracks)
-    def pose_tracks_params(self, request):
+    @pytest.fixture(params=position_arrays)
+    def position_arrays_params(self, request):
```

Comment:
```suggestion
    def position_array_params(self, request):
```
as it's returning params for a single position array at a time